esm-samples/jsapi-node/README.md

```diff
@@ -1,48 +1,47 @@
-# ArcGIS API for JavaScript in Node
+# ArcGIS API for JavaScript in Node.js
 
-Integrating Node with [`@arcgis/core`](https://www.npmjs.com/package/@arcgis/core) can be done by building the app with [native ESM in a supported Node version](https://nodejs.org/dist/latest-v14.x/docs/api/esm.html) or by transpiling to CommonJS.
+Integrating Node.js with [`@arcgis/core`](https://www.npmjs.com/package/@arcgis/core) can be done by building the app with [native ESM in a supported Node version](https://nodejs.org/dist/latest-v14.x/docs/api/esm.html) or by transpiling to CommonJS. This sample contains examples of both approaches.
+
+## Working with assets
+
+Working with the ArcGIS API for JavaScript's ES modules in Node.js requires configuring the APIs assets, since the application will be running on a local server and not in a web app. The assets include styles, images, fonts, and localization files. Be sure to set [`config.assetsPath`](https://developers.arcgis.com/javascript/latest/api-reference/esri-config.html#assetsPath) so that the assets are correctly resolved, for example:
```

Comment:
It may still work if you don't set `assetsPath`, but it'd be before for it to refer to the local assets than loading them from the CDN at runtime.

Reply:
I'm not sure what you mean? I tried it without setting `assetsPath` and it didn't work.

Reply:
Done.